src/test/java/io/openliberty/tools/intellij/it/TestUtils.java

```diff
@@ -576,7 +576,7 @@ public static void copyDirectory(String sourceDirectoryLocation, String destinat
      * This searches the output of this JUnit run for SocketTimeoutException which
      * has been identified as a fatal error and occurs during the Mac tests.
      */
-    public static synchronized void detectFatalError() {
+    public static void detectFatalError() {
```

Comment:
FYI to other reviewers. This is a response to my comment from a previous PR where after other changes were made, `synchronized` is no longer required on this method.

Reply:
Thank you!